pkg/kube/snapshot/snapshot_alpha.go

```diff
@@ -318,24 +317,23 @@ func (sna *SnapshotAlpha) CreateContentFromSource(ctx context.Context, source *S
 	return nil
 }
 
+func isReadyToUseAlpha(us *unstructured.Unstructured) (bool, error) {
+	vs := v1alpha1.VolumeSnapshot{}
+	if err := TransformUnstructured(us, &vs); err != nil {
+		return false, err
+	}
+	// Error can be set while waiting for creation
+	if vs.Status.Error != nil {
```

Comment:
Just wanted to confirm that the events are in `VolumeSnapshot` resource right, and not in the `VolumeSnapshotContent` resource?

Reply:
Yes. I don't think we read `VolumeSnapshotContent` status anywhere in our code. But status errors are propagated by snapshot_controller to `VolumeSnapshot` if there is an error in `VolumeSnapshotContent` as I understand.